demo/full/styles/style.css

```diff
@@ -1,40 +1,67 @@
-body {
-  margin: 0px;
-}
-
 @font-face {
   font-family: "icons";
   src: url("../fonts/FontAwesome.otf");
   font-weight: normal;
   font-style: normal;
 }
 
+@font-face {
+  font-family: "Roboto";
+  src: url("../fonts/Roboto-Regular.otf");
+  font-weight: normal;
+  font-style: normal;
+}
+
+body {
+  margin: 0px;
+  font-family: "Roboto";
+}
+
 .icon {
   font-family: "icons", sans-serif;
 }
 
+.headers-title {
+  display: flex;
+  align-items: center;
+}
+
 .title-wrapper h1 {
-  font-family: "Open Sans", sans-serif;
-  display: block;
+  display: inline-block;
   width: 100%;
   text-align: center;
   line-height: 1.05;
-  color: #555;
+}
+
+.title > * {
+  text-decoration: none !important;
+}
+
+.title-logo {
+  display: inline-block;
+  height: 25px;
+  display: inline-block;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Unexpected duplicate "display" ([css:S4656](https://sonarcloud.io/organizations/rx-player/rules#rule_key=css%3AS4656))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=312&issues=AWXYim4J_eEZPtUmD8V5&open=AWXYim4J_eEZPtUmD8V5)

[ISSUE_KEY]: AWXYim4J_eEZPtUmD8V5